feature/autodiscovery/providersxml/src/test/kotlin/app/k9mail/autodiscovery/providersxml/ProvidersXmlDiscoveryTest.kt

```diff
@@ -1,24 +1,32 @@
 package app.k9mail.autodiscovery.providersxml
 
 import androidx.test.core.app.ApplicationProvider
+import app.k9mail.core.android.testing.RobolectricTest
+import app.k9mail.core.common.oauth.OAuthConfiguration
+import app.k9mail.core.common.oauth.OAuthConfigurationProvider
 import assertk.assertThat
 import assertk.assertions.isEqualTo
 import assertk.assertions.isNotNull
 import assertk.assertions.isNull
-import com.fsck.k9.RobolectricTest
 import com.fsck.k9.mail.AuthType
 import com.fsck.k9.mail.ConnectionSecurity
-import com.fsck.k9.oauth.OAuthConfiguration
-import com.fsck.k9.oauth.OAuthConfigurationProvider
 import org.junit.Test
+import org.mockito.Mockito.mock
+import org.mockito.kotlin.doReturn
+import org.mockito.kotlin.stub
 
 class ProvidersXmlDiscoveryTest : RobolectricTest() {
     private val xmlProvider = ProvidersXmlProvider(ApplicationProvider.getApplicationContext())
-    private val oAuthConfigurationProvider = createOAuthConfigurationProvider()
+    private val oAuthConfigurationProvider = mock<OAuthConfigurationProvider>()
```

Comment:
Let's try to avoid using Mockito if easily possible.

Reply:
What is the state of Mockito in the project? So far we didn't talk about using or not using it.